src/manufacturer/manufacturerLandingPage.component.tsx

```diff
@@ -56,8 +68,25 @@ function ManufacturerLandingPage() {
             >
               Print
             </Button>
+            </Grid>
+
+            <Grid  sx={{ display: 'flex'}}>
+            <Button
+              sx={{ mx: 0.5 }}
+              variant="outlined"
+              startIcon={<RefreshIcon />}
+              onClick={() => {
+                handleRefresh();
+              }}
+            >
+              Refresh
+          </Button>
+          <Typography data-testid="refresh-timestamp" textAlign={'center'} variant="body1">
+              {formatDateTimeStrings(editRefreshDate.toISOString(), true)}
```

Comment:
Final comment! Just makes it a tad clearer what the actual timestamp is for
```suggestion
              Last refreshed at: {formatDateTimeStrings(editRefreshDate.toISOString(), true)}
```